### ISSUE TITLE
Make WoeUSB into Debian archive

### ISSUE THREAD

**Issue by Lin-Buo-Ren** [slacka/WoeUSB#127]
This issue is derived from [Support Launchpad PPA, openSUSE Build Service, or other software archive solutions · Issue #104 · slacka/WoeUSB](https://github.com/slacka/WoeUSB/issues/104).

Although WoeUSB has been Debianized, it has never being included into the official Debian archive(and other derived distributions based on it), by making this happen we can drop lots of effort to maintain an custom software archive.

This issue will track the progress of the inclusion.

**Comment by anthonyfok** [comment]
Hello!  Are you still looking for a sponsor uploading to Debian?
I am a Debian Developer.  :-)
感謝你們寫這個程式！今天剛好有需要，才發現有 WoeUSB 這麼實用的軟體存在！ :-)
Cheers,
Anthony 霍東靈

**Comment by Lin-Buo-Ren** [comment]
Yes we do need a sponsor.  As I'm currently focusing on packaging it to snappy, feel free to take over the task.
